src/khoj/routers/helpers.py

```diff
@@ -105,6 +109,15 @@ def update_telemetry_state(
     ]
 
 
+def construct_chat_history(conversation_history: dict, n: int = 4) -> str:
+    chat_history = ""
+    for chat in conversation_history.get("chat", [])[-n:]:
+        if chat["by"] == "khoj" and chat["intent"].get("type") == "remember":
```

Comment:
We should (at some point) set the intent type based on command used (e.g "online-information" vs "remember" vs "draw"). The schema hasn't aged well (wasn't great earlier either) but the point is to store the user intent inferred by Khoj for each chat message to respond appropriately